Makefile

```diff
@@ -34,6 +34,10 @@ run:
 run_local:
 	${DOCKER_COMPOSE_CMD} up -d redis postgres mailhog rabbitmq
 
+.PHONY: run_local_mac
+run_local_mac:
+	docker compose up -d redis postgres mailhog rabbitmq
+
```

Comment:
I have a more generic solution for this in "develop" branch, check it

Reply:
In any case this is essentially the same to `run_local`, and outside of the scope of this ticket